lib/game.rb

```diff
@@ -51,4 +53,13 @@ def print_board
     @display.output(@presenter.print_board(@board))
     @display.output(@presenter.print_results(end_game))
   end
+
+  def get_valid_input
+    input = @active_player.get_input(@display)
+    until @validator.valid_move?(@board, input)
+      @display.output(INPUT_ERROR)
+      input = @active_player.get_input(@display)
+    end
+    input
+  end
```

Comment:
Should get_valid_input be a responsibility of game or player?  The player gives the input, right?  The player should be giving a selection to the game, and the selection should be valid.